packages/code-studio/public/electron.js

```diff
@@ -1,26 +1,29 @@
-const electron = require('electron')
+const electron = require('electron');
```

Comment:
We could proooobably just delete this file... we don't at all.

Reply:
I'll delete it